js/ui/internetSearch.js

```diff
@@ -69,7 +69,7 @@ const InternetSearchProvider = new Lang.Class({
                 name = _("Open \"%s\" in browser").format(uri);
             } else if (resultId.startsWith('search:')) {
                 let query = resultId.slice('search:'.length);
-                name = _("Search Google for \"%s\"").format(query);
+                name = _("Search the web for \"%s\"").format(query);
```

Comment:
I'm not sure if we want this string change.  In the past I recall Matt being pretty specific about how he wanted to present the search results.  Another easy option for now would be to leave "Google" in the English string and simply edit the Chinese translation to replace "Google" with "Baidu".